cmd/nerdctl/stats_linux.go

```diff
@@ -62,11 +62,13 @@ func setContainerStatsAndRenderStatsEntry(previousStats *statsutil.ContainerStat
 		if err != nil {
 			return statsutil.StatsEntry{}, fmt.Errorf("failed to retrieve the statistics in netns %s: %v", ns, err)
 		}
+		defer ns.Close()
```

Comment:
Can we wrap this into a function and log errors if Close returns an error; that would help us catch file leaks if there are any if still remaining? Ideally, this should never happen.